docs/source/index.rst

```diff
@@ -134,6 +155,7 @@ Welcome to AiiDA's documentation!
 .. toctree::
    :maxdepth: 2
    :caption: Internal architecture
+   :hidden:
```

Comment:
further down in the "acknowledgements", the bullet point list is not displayed as bullet points.

Could you please fix this as well?

Reply:
fixed